docker-compose.server.yml

```diff
@@ -16,7 +16,7 @@ services:
       - app
     environment:
       - PHP_MAX_EXECUTION_TIME=30
-      - PHP_MEMORY_LIMIT=128M
+      - PHP_MEMORY_LIMIT=512M
```

Comment:
What is the reason behind this increase in memory requirements. Even DDB CMS can run under 256 MB memory. If this is really an requirement I think we should analyse the code and figure out why.